src/sourmash/minhash.py

```diff
@@ -683,6 +704,26 @@ def contained_by(self, other, downsample=False):
 
         return self.count_common(other, downsample) / len(self)
 
+    def containment_ani(self, other, downsample=False, containment=None, confidence=0.95, return_ci = False):
+        "Estimate ANI from containment with the other MinHash."
+        self_mh = self
+        other_mh = other
+        scaled = self.scaled
+        if downsample:
+            scaled = max(self_mh.scaled, other_mh.scaled)
+            self_mh = self.downsample(scaled=scaled)
+            other_mh = other.downsample(scaled=scaled)
+        if containment is None:
+            containment = self_mh.contained_by(other_mh)
+        n_kmers = len(self_mh) * scaled # would be better if hll estimate - see #1798
+
+        ## ani_info will be two values if return_ci is False; four values if return_ci is True
+        ## not ideal, but I'm not sure how to fix.
+        ani_info = containment_to_distance(containment, self_mh.ksize, self_mh.scaled,
```

Comment:
I don't think you need to do it here, but you _could_ start using namedtuples or other data-only classes to return a more complex object type rather than four values.